composer.json

```diff
@@ -1,5 +1,6 @@
 {
-    "require": {
-        "phpunit/phpunit": "5.7.*"
+    "require-dev": {
+        "phpunit/phpunit": "^5.7.21 || ^6.5 || ^7.5",
```

Comment:
what's the logic behind these specific versions?

Reply:
These constraints are the 3 different major versions that are currently supported by WordPress.